infra/terraform/test-org/org/external_data/bar.csv

```diff
@@ -1,3 +1,3 @@
 id,name,dept
 3,Amy,SC
-4,Jacob,SC
\ No newline at end of file
+4,Jacob,SC

```

Comment:
Likely unimportant, but this will force a replace of `google_storage_bucket_object.ci_bq_external_hive_file_bar`